.gitignore

```diff
@@ -18,4 +18,3 @@ src/config/appConfig.js
 TODO
 .vscode
 .history
-
```

Comment:
Please, leave the blank line at EOF

Reply:
The newline at end of file is still there, that one is just an additional one :smiley:

Reply:
aaa ok